helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -75,9 +75,16 @@ internal class ListLicensesCommand : CliktCommand(
 
     private val onlyOffending by option(
         "--only-offending",
-        help = "Only list licenses causing a rule violation of error severity in the given ORT result."
+        help = "Only list licenses causing a rule violation of severity specified severity, see --severity."
     ).flag()
 
+    private val severity by option(
+        "--severity",
+        help = "Set the severities to use filtering enabled by --only-offending, specified as comma separated " +
+                "values. Allowed values: ERROR|WARNING|HINT."
+    ).convert { value -> value.split(",").map { Severity.valueOf(it) }.toSet() }
+        .default(setOf(*Severity.values()))
```

Comment:
Comments from the previous commit also apply here.

Reply:
Done.